R/analyze_binary.R

```diff
@@ -29,10 +29,10 @@ deepstate_harness_analyze_pkg <- function(path,testfiles="all",max_inputs="all")
       test.files <- test.files[1:testfiles]
     }
     for(pkg.i in seq_along(test.files)){
-      list_testfiles[basename(test.files[[pkg.i]])] <- list(deepstate_analyze_fun(path,basename(test.files[[pkg.i]]),max_inputs))
+      list_testfiles[basename(test.files[[pkg.i]])] <- list(deepstate_analyze_fun(path,basename(test.files[[pkg.i]]), max_inputs, verbose))
```

Comment:
please use double square brackets on left side of <- and remove list() from right side

Reply:
I solved this in the commit 1d7e9cc